src/Lib/Route/RouteScanner.php

```diff
@@ -99,7 +99,7 @@ private function loadRoutes(): void
                 return strstr($fqn, '\\' . $prefix . $controller . 'Controller');
             });
 
-            if (count($results) == 1) {
+            if (count($results) >= 1) {
```

Comment:
Why was this change necessary?

Reply:
For some reason, when using the cli commands and adding the namespaces['controllers'] setting, the config is parsed twice, resulting in 2 identical entries in $namespaces['controllers'] which leads to 2 identical entries in the $result array.

Reply:
Can you paste in your `swagger_bake.php` config?

Reply:
```php
return [
    'SwaggerBake' => [
        'prefix' => '/api-v3',
        'yml' => '/config/swagger.yml',
        'json' => '/webroot/swagger.json',
        'webPath' => '/swagger.json',
        'hotReload' => true, //\Cake\Core\Configure::read('debug'),

        'namespaces' => [
            'controllers' => ['\ApiV3\\'],
            'entities' => ['\SharedModels3\\'],
            'tables' => ['\SharedModels3\\']
        ]
    ]
];
```

Reply:
Reference this pull request I just opened: https://github.com/cnizzardini/cakephp-swagger-bake/pull/276

I think it likely solves these issues, but I wasn't exactly sure how to push into your PR or fork. That's beyond my gitfu.

Reply:
Yes, I tested that pull request and it works as expected.

Reply:
Okay, we'll close this PR out and use that one. I should be able to get a release out in the next day or two after I regression test a bit more.